x-pack/test/functional/apps/dashboard/dashboard_maps_by_value.ts

```diff
@@ -77,7 +78,16 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
   describe('dashboard maps by value', function () {
     before(async () => {
       await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/logstash_functional');
-      await esArchiver.loadIfNeeded('x-pack/test/functional/es_archives/lens/basic');
+      await kibanaServer.importExport.load(
+        'x-pack/test/functional/fixtures/kbn_archiver/lens/lens_basic.json'
+      );
+    });
+
+    after(async () => {
+      await esArchiver.unload('x-pack/test/functional/es_archives/logstash_functional');
+      await kibanaServer.importExport.unload(
+        'x-pack/test/functional/fixtures/kbn_archiver/lens/lens_basic.json'
```

Comment:
I'm having a little bit of a hard time telling if this test is creating new saved objects or if it's only doing "by value" in a dashboard and not saving it?  If it is saving new saved objects we should also use the `clean` method to cleanup.